ingestion_server/ingestion_server/cleanup.py

```diff
@@ -383,7 +451,8 @@ def clean_image_data(table):
     conn.commit()
     iter_cur.close()
     conn.close()
-    end_time = time.time()
+    data_uploader.flush()
+    end_time = time.perf_counter()
```

Comment:
Nit: it would be nice to format `cleanup_time` in the log: `{cleanup_time:.3f}`

Reply:
Done.